rest.go

```diff
@@ -75,6 +78,33 @@ func getBaseUrl() string {
 	return DEFAULT_BASE_URL
 }
 
+func structToURLValues(input interface{}) string {
+	queryValues := url.Values{}
+	structType := reflect.TypeOf(input)
+	structValue := reflect.ValueOf(input)
+
+	for i := 0; i < structType.NumField(); i++ {
+		field := structType.Field(i)
+		value := structValue.Field(i)
+
+		if !value.IsNil() {
+			queryParam := field.Tag.Get("queryParam")
+			if queryParam == "" {
+				queryParam = field.Name
+			}
+			val := value.Elem().Interface()
+
+			if strVal, ok := val.(string); ok {
```

Comment:
I think this case is not needed and you can just use `queryValues.Add(queryParam, fmt.Sprintf("%v", val))` for strings too.
It's true it's not needed to use `Sprintf` there but it would make the code simpler IMHO

Reply:
Done